pages/levels/5.vue

```diff
@@ -0,0 +1,89 @@
+
+But why is it so important to use the right element?
+Firstly, it makes it easier for people to navigate with a keyboard or screen reader if the technology can recognise what it’s navigating to. Secondly, it’s also smaller in file size and easier to make responsive when developing for mobile phones. Lastly, why would you make an effort to make everything accessible, but then not use the built-in accessibility from plain HTML elements?
+## Links
+- [HTML and accessibility](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML)`,
+
+      code,
+      defaultCode: code,
+      task: "Help - I can’t navigate!",
+      coloredButton: false,
+    };
+  },
+  computed: {
+    isCorrect(): boolean {
+      return /<button>\s*Click me!\s*<\/button>/.test(
+        this.code,
+      );
+    },
+  },
+  methods: {
+    switchButtonColor() {
+      this.coloredButton = !this.coloredButton;
+    },
+  },
+});
+</script>
+
+<template>
+  <Level :markdown :task :isCorrect :levelNumber="5">
+    <div class="flex h-full w-full flex-col gap-8">
+      <div
+        class="flex flex-1 flex-row gap-8 rounded-2xl border-2 border-blue-4 bg-blue-3-light p-8 pt-6 shadow-small-drop-shadow *:flex *:flex-1 *:flex-col *:items-center *:gap-4 dark:bg-blue-3-dark"
+      >
+        <div title="Wrong use of HTML elements">
+          <Icon name="dashicons:no" size="2rem" />
+          <div
+            class="grid flex-1 place-items-center self-stretch text-pretty rounded-2xl bg-blue-5-light p-4 text-center shadow-small-drop-shadow dark:bg-blue-5-dark forced-colors:outline"
+          >
+            <div
+              @click="switchButtonColor"
+              class="truncate rounded-xl border-2 border-text-light p-4 text-xl dark:border-text-dark"
+              :class="{
+                'bg-axolotl-light': coloredButton,
+                'dark:bg-axolotl-dark': coloredButton,
+              }"
+            >
+              Click me!
+            </div>
+          </div>
+        </div>
+        <div title="Intended use of HTML elements">
+          <Icon name="dashicons:yes" size="2rem" />
+          <div
+            class="grid flex-1 place-items-center text-center self-stretch rounded-2xl bg-blue-5-light p-4 shadow-small-drop-shadow dark:bg-blue-5-dark forced-colors:outline"
+          >
+            <div
+              v-html="code"
+              class="w-full truncate text-xl [&_button]:p-4 [&_button]:rounded-xl [&_button]:border-2 [&_button]:border-text-light [&_button]:dark:border-text-dark"
+              :class="{
+                '[&_button]:bg-axolotl-light': coloredButton,
+                '[&_button]:dark:bg-axolotl-dark': coloredButton,
+              }" @click="coloredButton = !coloredButton"
+            ></div>
+          </div>
+        </div>
+      </div>
+      <div>
+        <SingleLineCodeEditor
+          before='<body>
+    <div @click= "changeColor">Click me!</div>',
```

Comment:
```suggestion
    <div @click="changeColor">Click me!</div>',
```
da ist ein Abstand zu viel